devolving_music/lib/song_scores.py

```diff
@@ -110,37 +123,72 @@ def get_quality_list(self,
 
         return final_quality_list
 
-    ###
-    # static methods can be moved into helper function
-    ###
     @staticmethod
     # sorts in ascending order
     def get_info_sort(
-            score_suite_list: List["ScoreSuite"]) -> List["ScoreSuite"]:
+            score_suite_list: List["ScoreSuite"],
+            get_informed=False,
+            informed_measure=1) -> List["ScoreSuite"]:
         """
-        Returns a list of song suites ordered by low information songs with the lowest information score 
-        are randomly shuffled before being added to the list
-        
+        Returns a list of song suites ordered by low information songs with the lowest information score
+        randomly shuffled before being added to the list
+
         """
         score_suite_list.sort(key=lambda sub: sub.info_score)
```

Comment:
```suggestion
        score_suite_list = sorted(score_suite_list, key=lambda sub: sub.info_score)
```

As is, it sorts the list in place. Function should avoid mutating their arguments.

Reply:
Yes but it's slightly faster to do this way and it saves memory.

Reply:
https://pythonsimplified.com/difference-between-sort-and-sorted-in-python/#Which_is_faster_sort_or_sorted

Reply:
![image](https://user-images.githubusercontent.com/22030302/186464556-caee0e84-e3fd-480f-b3f8-b57b8cfc9002.png)

Reply:
Okay fine. But this is not a diff for this PR. I'll have to refactor and change .sort to sorted for all the sorting functions and verify that things don't break. I made a new issue and I'll do this on a new pull request. I don't think it is relevant for this one.